### PR TITLE
ON-41251 # Added state and territories to conditional address

### PR DESCRIPTION
**Pull request by jdawg093**

### CODE REVIEW

**Comment by divporter** [comment]
May as well type the filter settings too.

**Comment by jdawg093** [comment]
I assume that'll be more code changes in other repos because of the types, we happy with that?

**Comment by divporter** [comment]
I'm already doing it in validation, so the types should match.

File: typescript/point/index.d.ts [hunk 1]
```diff
@@ -3,3 +3,14 @@ import { Addresses } from './addresses'
...
+
+export type PointStatesAndTerritories =
+  | 'NSW'
+  | 'QLD'
+  | 'VIC'
+  | 'ACT'
+  | 'TAS'
+  | 'SA'
+  | 'NT'
+  | 'WA'
```
Comment (divporter) [inline typescript/point/index.d.ts:15]: perhaps this could be a common union type + 'OT'

**Pull request merged**